src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -112,15 +112,19 @@
     <ItemGroup> 
       <!-- Resolve all package references to their physical location first -->
       <_ResolvedPackageReference Include="%(PackageReference.Identity)">
-        <PhysicalLocation>$(NuGetPackageRoot)%(PackageReference.Identity)\%(PackageReference.Version)</PhysicalLocation>
+        <PhysicalLocation>$(NuGetPackageRoot)%(PackageReference.Identity)/%(PackageReference.Version)</PhysicalLocation>
       </_ResolvedPackageReference>
+      <_ResolvedDacpacFile Include="@(_ResolvedPackageReference)">
+        <DacpacFile>$([System.String]::new('%(PhysicalLocation)/tools/').ToLower())%(Identity).dacpac</DacpacFile>
+      </_ResolvedDacpacFile>
```

Comment:
I think we can actually simplify this back to a single ItemGroup again. Something like this should work:

```
$(NuGetPackageRoot)$([System.String]::new('%(PackageReference.Identity)/%(PackageReference.Version)').ToLower())/tools/%(Identity).dacpac
```

Reply:
Done. My version is slightly different, as I believe we want `$NuGetPackageRoot)` inside the string function.